scheduler/src/cook/util.clj

```diff
@@ -24,7 +24,8 @@
             [clojure.tools.namespace.dependency :refer :all]
             [postal.core :as postal]
             [schema.core :as s])
-  (:import (java.util.concurrent.atomic AtomicLong)))
+  (:import (java.util.concurrent.atomic AtomicLong)
```

Comment:
Why AtomicLong?

Reply:
No idea; the `AtomicLong` was already there, this is unchanged